packages/gatsby-theme/src/blocks/ShareBlock/index.js

```diff
@@ -0,0 +1,76 @@
+import PropTypes from "prop-types"
+import { Box } from "theme-ui"
+import Share from "../../components/Share"
+
+const ShareBlock = ({ heading, config, disabled, ...props }) => {
+  const generalConfig = { title: config.title, to: config.to }
+
+  return (
+    <Box variant='wrapper' __themeKey='container'>
+      <Share {...props}>
+        {heading && <Share.Heading>{heading}</Share.Heading>}
+        <Share.Container>
+          {!disabled.facebook && (
+            <Share.Facebook {...generalConfig} {...config.facebook} />
+          )}
+          {!disabled.linkedin && (
+            <Share.Linkedin {...generalConfig} {...config.linkedin} />
+          )}
+          {!disabled.reddit && (
+            <Share.Reddit {...generalConfig} {...config.reddit} />
+          )}
+          {!disabled.twitter && (
+            <Share.Twitter {...generalConfig} {...config.twitter} />
+          )}
+          {!disabled.whatsapp && (
+            <Share.Whatsapp {...generalConfig} {...config.whatsapp} />
+          )}
```

Comment:
We should also verify if the config exists in the conditional. 
```
{!disabled.facebook && config.facebook && (
```
We also should reconsider the need for using the `disabled` property, since they could be "disabled" by not sending specific property in the `config` prop.